<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FemGui::DlgSettingsFemInOutVtk</class>
 <widget class="QWidget" name="FemGui::DlgSettingsFemInOutVtk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VTK</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="gdl_import">
   <item>
    <widget class="QGroupBox" name="gpb_import">
     <property name="title">
      <string>Import</string>
     </property>
     <layout class="QGridLayout" name="girdLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Which object to import into</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Gui::PrefComboBox" name="cb_import_vtk">
        <property name="toolTip">
         <string>VTK result object: A FreeCAD FEM VTK result object will be imported
(equals to the object which was exported).

FEM mesh object: The results in the VTK file will be omitted, only the
mesh data will be imported and a FreeCAD FEM mesh object will be created.

FreeCAD result object: The imported data will be converted into a
FreeCAD FEM Result object. Note: this setting needs the exact result
component names and thus it only works properly with VTK files
exported from FreeCAD.</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>Choose in which object to import into</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="prefEntry" stdset="0">
         <cstring>ImportObject</cstring>
        </property>
        <property name="prefPath" stdset="0">
         <cstring>Mod/Fem/InOutVtk</cstring>
        </property>
        <item>
         <property name="text">
          <string>VTK result object</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FEM mesh object</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FreeCAD result object</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gpb_export">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QGridLayout" name="gdl_export">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_fem_mesh">
        <property name="text">
         <string>FEM Mesh export elements</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Gui::PrefComboBox" name="cb_export_fem_mesh">
        <property name="toolTip">
         <string>Chose which elements export</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="prefEntry" stdset="0">
         <cstring>ExportFemMesh</cstring>
        </property>
        <property name="prefPath" stdset="0">
         <cstring>Mod/Fem/InOutVtk</cstring>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Higher dimension</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
